### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by PaulStorberg**

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -33,12 +34,11 @@
       </div>
     </div>
   </div>
-  <a id="services" class="anchor"></a>
   <div class="row works-container">
```
Comment (kevinstorberg) [inline app/views/home/index.html.erb:37]: Why is this called works-container but the anchor is called work?

File: app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -15,9 +15,26 @@
       <li class="nav-item">
         <a class="nav-link" href="#contact">Hire Us</a>
       </li>
+      <li class="nav-item">
+        <div class="dropdown">
+          <div class="btn btn-secondary dropdown-toggle" style="margin-top: 8px;" type="button" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Sign In
+          </div>
+          <form class="dropdown-menu p-4">
```
Comment (kevinstorberg) [inline app/views/layouts/_navbar.html.erb]: Remove this dropdown please

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -10,7 +10,8 @@
   </head>
 
   <body style="height: 100%; position: relative; margin-top: 50px; font-family: Playfair display;">
```
Comment (kevinstorberg) [inline app/views/layouts/application.html.erb]: Lets move this style to the CSS files

File: app/views/portfolio_items/_form.html.erb [hunk 4]
```diff
@@ -1,31 +1,35 @@
-<!-- <div class="col-xs-12 col-md-6 col-md-offset-3">
-  <div class="form-container">
-    <%= form_for @portfolio_item remote: true do |f| %>
+<td colspan="6">
+  <div id="portfolio_itemForm">
```
Comment (kevinstorberg) [inline app/views/portfolio_items/_form.html.erb:2]: It is sort of weird to switch from snake case to camel case. You should stick with only one. CSS generally uses dashes.

File: app/models/dashboard.rb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+class Dashboard < ApplicationRecord
```
Comment (kevinstorberg) [inline app/models/dashboard.rb]: Get rid of this.

File: app/controllers/users_controller.rb [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+  private
+
+  def set_user
+    @user = params[:id].present? ? User.find(params[:id]) : User.new
+  end
+
+  def user_params
+    params.require(:user).permit(
+      :id,
```
Comment (kevinstorberg) [inline app/controllers/users_controller.rb]: Alphabetize these

File: app/views/home/index.html.erb (hunk 1 above)
Comment (PaulStorberg) [reply app/views/home/index.html.erb:37]: To match the navbar button

File: app/views/layouts/_navbar.html.erb (hunk 2 above)
Comment (PaulStorberg) [reply app/views/layouts/_navbar.html.erb]: complete

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (PaulStorberg) [reply app/views/layouts/application.html.erb]: complete

File: app/views/portfolio_items/_form.html.erb (hunk 4 above)
Comment (PaulStorberg) [reply app/views/portfolio_items/_form.html.erb:2]: complete

File: app/models/dashboard.rb (hunk 5 above)
Comment (PaulStorberg) [reply app/models/dashboard.rb]: complete

File: app/controllers/users_controller.rb (hunk 6 above)
Comment (PaulStorberg) [reply app/controllers/users_controller.rb]: complete

**Pull request merged**